components/inputs/input-date.js

```diff
@@ -165,7 +170,7 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 					@keydown="${this._handleKeydown}"
 					label="${ifDefined(this.label)}"
 					?label-hidden="${this.labelHidden}"
-					placeholder="${this._dateTimeDescriptor.formats.dateFormats.short}"
+					placeholder="${(this._dateTimeDescriptor.formats.dateFormats.short).toUpperCase()}"
```

Comment:
Jeff requested the placeholder be uppercase (e.g., instead of "M/d/yyyy" have it be "M/D/YYYY") for a more consistent look. I'll confirm that this doesn't result in screenreader yelling.

Reply:
Cool. The case in the actual date format strings actually means something, but in this context it should be OK to uppercase it.

Reply:
yeah we were thinking/hoping users wouldn't necessarily know/care about that.

Reply:
Checked NVDA with firefox and Voice over with Safari and did not experience any yelling with all caps.

Reply:
Other option would be to use `text-transform: uppercase` in the CSS. If this work though, 🤷‍♂